### PR TITLE
On branch edburns-msft-dd-1384346-marketplace-entry Copy from tWAS, but update README

### PR DESCRIPTION
**Pull request by edburns**
…u
new file:   src/main/resources/README.md
new file:   src/main/resources/marketing-artifacts/Large_216_216.png
new file:   src/main/resources/marketing-artifacts/Medium_90_90.png
new file:   src/main/resources/marketing-artifacts/Small_48_48.png
new file:   src/main/resources/marketing-artifacts/Wide_255_115.png

Signed-off-by: Ed Burns <edburns@microsoft.com>

### CODE REVIEW

**Comment by edburns** [comment]
Seeking review from @gcharters and Shane O'Rourke.

File: src/main/resources/README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+### Lead management
+
+- None (TBD)
+
+## Properties
+
+### Category
+
+- Application Infrastructure
+- Client Operating System
```
Comment (m-reza-rahman) [inline src/main/resources/README.md:30]: Is this really a "client operating system"? I think the rest is fine.

File: src/main/resources/README.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  - IBM WebSphere Liberty provides a flexible, secure server runtime environment for large-scale and mission critical application deployments. This offer provisions an IBM WebSphere Liberty environment on Azure Red Hat OpenShift.
+
+### Legal Links
+
+- Privacy policy link
+  - https://www.ibm.com/privacy/us/en/ ???
+
+### Useful Links
+
+- [Overview](https://openliberty.io/)
```
Comment (m-reza-rahman) [inline src/main/resources/README.md:62]: Are these the right links for WebSphere Liberty? Also, should the offer handle both Open Liberty and WebSphere Liberty?

File: src/main/resources/README.md (hunk 1 above)
Comment (edburns) [reply src/main/resources/README.md:30]: Removed.

File: src/main/resources/README.md (hunk 2 above)
Comment (edburns) [reply src/main/resources/README.md:62]: I have filed https://github.com/WASdev/azure.liberty.aro/issues/10

**Review by edburns — COMMENTED** [review]
I have applied the suggested changes from @m-reza-rahman .

**Review by shane-orourke — APPROVED** [review]

**Pull request merged**